docs/development/local_setup.md

```diff
@@ -1,65 +1,101 @@
-
-```bash
-$ brew install dep
-```
-
 ## Installing `kubectl` and `helm`
-As already mentioned in the introduction, the communication with the Gardener happens via the Kubernetes (Garden) cluster it is targetting. To interact with that cluster, you need to install `kubectl`:
+As already mentioned in the introduction, the communication with the Gardener happens via the Kubernetes (Garden) cluster it is targetting. To interact with that cluster, you need to install `kubectl`.
+
+On Mac OS run
 
 ```bash
 $ brew install kubernetes-cli
 ```
+Please check the [kubectl installation documentation](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for other OS.
 
 You may also need to develop Helm charts or interact with Tiller using the [Helm](https://github.com/kubernetes/helm) CLI:
 
+On Mac OS run
+
 ```bash
 $ brew install kubernetes-helm
 ```
 
+On other OS please check the [Helm installation documentation](https://github.com/kubernetes/helm/blob/master/docs/install.md).
+
 ## Installing `git`
-We use `git` as VCS which you need to install:
+We use `git` as VCS which you need to install.
+
+On Mac OS run
 
 ```bash
 $ brew install git
 ```
 
-## Installing Docker (Optional)
+On other OS, please check the [Git installation documentation](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+## Installing Minikube
+
+You'll need to have [minikube](https://github.com/kubernetes/minikube#installation) installed and running.
+
+## [Optional] Installing Docker
 In case you want to build Docker images for the Gardener you have to install Docker itself. We recommend using [Docker for Mac OS X](https://docs.docker.com/docker-for-mac/) which can be downloaded from [here](https://download.docker.com/mac/stable/Docker.dmg).
 
-## Installing `gcloud` SDK (Optional)
-In case you have to create a new release or a new hotfix of the Gardener you have to push the resulting Docker image into a Docker registry. Currently, we are using the Google Container Registry (this could change in the future). Please follow the offical installation instructions [from Google](https://cloud.google.com/sdk/downloads#mac).
+On other OS, please check the [Docker installation documentation](https://docs.docker.com/install/).
+
+## [Optional] Installing `gcloud` SDK
+In case you have to create a new release or a new hotfix of the Gardener you have to push the resulting Docker image into a Docker registry. Currently, we are using the Google Container Registry (this could change in the future). Please follow the offical [installation instructions from Google](https://cloud.google.com/sdk/downloads).
+
 
+# Local Gardener only setup
+
+This setup is only ment to be used for developing purposes, which means that _not_ all three Gardener Kubernetes clusters, the _Gardener_, the _Seed_ and finally the _Shoot_ clusters are installed but only the control plane of the Gardener cluster.
```

Comment:
*ment=meant

Can you replace the underlines with proper markdown directives?
Moreover, I find the description kind of confusing: "all three Gardener Kubernetes clusters" ???